src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import SignUp from './components/Register/SignUp';
+import SignIn from './components/SignIn/SignIn';
+import Dashboard from './components/Dashboard/Dashboard';
+import Home from './components/Lease/Home';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route
+} from 'react-router-dom';
```

Comment:
for readability, some good conventions are importing in the following order
'react'
libraries
local src
scss

Reply:
fixed